tests/integration_cases/test_raw_node.rs

```diff
@@ -111,6 +111,53 @@ fn test_raw_node_step() {
     }
 }
 
+// test_raw_node_committed_entries_pagination ensures that max_committed_size_per_ready
+// cap the size of committed entries in each ready
+#[test]
+fn test_raw_node_committed_entries_pagination() {
+    setup_for_test();
+    let mut cfg = new_test_config(1, vec![], 10, 1);
+    cfg.max_committed_size_per_ready = 1024;
+    let storage = new_storage();
+    let mut raw_node = RawNode::new(&cfg, storage.clone(), vec![new_peer(1)]).unwrap();
+    let rd = raw_node.ready();
```

Comment:
This three lines seem unnecessary.

Reply:
Do you mean the 3rd line or all the lines from 121 to 123? I couldn't figure it out why the `raw_node` should be removed.